operator/pkg/helmreconciler/prune.go

```diff
@@ -139,9 +139,11 @@ func (h *HelmReconciler) DeleteObjectsList(objectsList []*unstructured.Unstructu
 				continue
 			}
 
-			err := h.client.Delete(context.TODO(), &o, client.PropagationPolicy(metav1.DeletePropagationBackground))
+			err := h.client.Delete(context.TODO(), &o,
+				client.PropagationPolicy(metav1.DeletePropagationBackground), client.GracePeriodSeconds(0))
```

Comment:
GracePeriodSeconds=0 may have some negative impacts, what is it needed for? we should at the very least comment in the code what its there for

Reply:
Especially since it doesn't seem to solve the root problem of IOP finalizer anyways? What is it intended for?

Reply:
yep, just added for testing before, removed